Makefile

```diff
@@ -27,15 +27,20 @@ endif
 ##@ Default target
 all: gitops ## Install dependencies and build Gitops binary
 
+
+TEST_TO_RUN=./...
```

Comment:
Is the intent of this variable that people can change it to override which unit tests they’re running? If so, would it make sense to assign it with `?=`, so that it’s possible to override it on the command line? That solution seems to work for me, but I’m no Make expert, so I might have missed something.

If it’s intended for something else, what is that? I’m struggling to figure it out.

If it’s a remnant of trying out a different approach, how about we fold it back into the `unit-tests` command for consistency’s sake?

Reply:
Right. Will do.

Somewhat pointless context:

I wrote these patches on top of some of my other PRs to enable more tests. I've used it as `make unit-tests`, wait for the test to finish, and then copy a failing suit name and ploink it at the end, e.g. `make unit-tests TEST_TO_RUN=./pkg/server` - this works just fine, as explicit make variables can override defaults with `=`.

However `TEST_TO_RUN=./pkg/server make unit-tests` doesn't work now. Allowing that too would be more helpful. I think `?=` is a GNU Make-ism, but we don't to my knowledge have any passionate OpenBSD people, so that's fine.

Reply:
That was super helpful context; thank you. Not pointless at all. 😄